demo/button/basic/playground

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable no-console, no-undef, react/prop-types, react/react-in-jsx-scope, react/jsx-no-undef */
+
+const capitalize = (s) => s[0].toUpperCase() + s.slice(1, s.length)
+
+const layouts = ['full']
+const types = ['primary', 'secondary', 'accent', 'ghost', 'flat']
+const sizes = ['small', 'medium', 'large']
+
+const Icon = (
```

Comment:
As far as I know, there's no way to import nor require a library/component in the playground. Maybe @carlosvillu could point me about how to do that.